var/spack/repos/builtin/packages/hepmc/package.py

```diff
@@ -28,24 +28,34 @@ class Hepmc(CMakePackage):
 
     variant('python', default=False, description='Enable Python bindings')
     variant('rootio', default=False, description='Enable ROOT I/O')
+    variant('interfaces', default=False, description='Install interfaces for some Monte-Carlo Event Gens')
 
     depends_on('cmake@2.8.9:', type='build')
     # FIXME: Officially supports Python3, but the build system doesn't find it
-    depends_on('python@:2.99.99', when='+python')
+    #depends_on('python@:2.99.99', when='+python')
```

Comment:
Why is this commented out?

Reply:
I have a few corrections in local repo, will push soon.